server/test/write.go

```diff
@@ -363,28 +363,6 @@ var writeCommandTests = []writeCommandTest{
 		// output
 		err: serverErrors.DuplicateTupleInWrite(tk),
 	},
-	{
-		_name: "ExecuteDeleteTupleWhichDoesNotExistReturnsError",
```

Comment:
do we need to delete the test? can we not assert that it doesn't return an error?

Reply:
Good point. I'll do that. Thanks!